src/co/elastic/NotificationManager.groovy

```diff
@@ -61,11 +65,45 @@ def analyzeFlakey(Map params = [:]) {
         }
       }
     }
-    def msg = "❄️ The following tests failed but also have a history of flakiness and may not be related to this change: " + ret.join("\n")
-    
-    if (ret) {
-      githubPrComment(message: msg, commentFile: 'flakey.id')
+
+    def labels = 'flaky-test,ci-reported'
+    def tests = lookForGitHubIssues(flakeyList: ret, labelsFilter: labels.split(','))
```

Comment:
We are mixing two spellings here. We have `flakey` and `flaky`. I think we should standarize on `flaky` wherever possible.